### PR TITLE
Heading adjustments

### PR DESCRIPTION
**Pull request by evoX0**
#### Please tick a box ###
- [ ] **feature** _(A new feature_)
- [ ] **fix** _(A bug fix_)
- [x] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [ ] Yes
- [x] No

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [x] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/Heading.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const MediumHeading = ProtoHeading.extend`
 	${props => getSizes({ size: 'medium', ...props })}
 
 	@media screen and (min-width: ${props => props.theme.variables.largeWidth}) {
-		${props => getSizes({ size: 'regular', ...props })}
+		${props => getSizes({ size: props.xsize || 'regular', ...props })}
```
Comment (misund) [inline src/atoms/Heading.js]: What is your intention here? I don't get it.

File: src/atoms/Heading.js [hunk 2]
```diff
@@ -81,12 +81,12 @@ const HugeHeading = ProtoHeading.extend`
...
 `;
 
-const Heading = ({ size, ...rest }) => {
+const Heading = ({ size, xsize, ...rest }) => {
 	switch (size) {
 	case 'small':
 		return <SmallHeading {...rest} />;
 	case 'medium':
-		return <MediumHeading {...rest} />;
+		return <MediumHeading xsize={xsize} {...rest} />;
```
Comment (misund) [inline src/atoms/Heading.js]: The changes on line 84 and 89 are redundant. You can remove them without having any effect on the behavior.

File: src/index.js [hunk 3]
```diff
@@ -125,6 +125,10 @@ export { ButtonWithModal         } from './atoms/ButtonWithModal';
 export { NumberedListItem        } from './atoms/NumberedListItem';
 
 
+export { ActivityTime         } from './atoms/SvgIcon/ActivityTime';
+export { Difficulty         } from './atoms/SvgIcon/Difficulty';
+export { TotalTime         } from './atoms/SvgIcon/TotalTime';
```
Comment (misund) [inline src/index.js]: Expose these via SvgIcon instead.

File: src/atoms/Heading.js (hunk 1 above)
Comment (misund) [reply src/atoms/Heading.js]: * What does xsize mean?
* Why do we break the pattern in the rest of the file?

**Comment by misund** [comment]
Please fill out the form in the PR description.

**Comment by evoX0** [comment]
I remove the part with Heading changes.
Just leave the export statement

**Comment by Fumler** [comment]
Is this still being worked on? @evoX0

**Pull request closed without merging**